cmd/rb-main.go

```diff
@@ -236,6 +236,13 @@ func mainRemoveBucket(ctx *cli.Context) error {
 			fatalIf(errDummy().Trace(), "`"+targetURL+"` is not empty. Retry this command with ‘--force’ flag if you want to remove `"+targetURL+"` and all its contents")
 		}
 		e := deleteBucket(targetURL)
+
+		if e == nil && !isNamespaceRemoval(targetURL) {
```

Comment:
It is easier to read the code if this block comes after the next fatalIf()

Reply:
done.

Reply:
You don't need to check for e == nil here, since it is obvious at this stage (you have fatalIf() before this line)

Reply:
removed.